tests/boards/intel_s1000_crb/src/gpio_test.c

```diff
@@ -122,13 +120,20 @@ void gpio_thread(void *dummy1, void *dummy2, void *dummy3)
 			toggle = 1;
 		}
 
-		ret = gpio_pin_write(gpio_dev, GPIO_OUT_PIN, toggle);
+		ret = gpio_pin_set(gpio_dev, GPIO_OUT_PIN, toggle);
```

Comment:
We can replace this with `gpio_pin_toggle`

Reply:
Updated.